tests/test_connection.py

```diff
@@ -19,43 +19,49 @@
 from nebula3.common import ttypes
 from nebula3.Exception import IOErrorException
 
+AddrIp = ['127.0.0.1', '::1']
+port = 29562
```

Comment:
Why we are using this port number, please?

Reply:
I forgot to change it back to the default port. This is the port of my local service.